server/src/domain/usecases/createTask.test.ts

```diff
@@ -0,0 +1,44 @@
+import { TaskRepository } from '../../infrastructure/persistance/taskRepository';
```

Comment:
The tests should be in a separate directory next to `src`.

Reply:
I put the test directory outside src folder